apps/web/src/pages/User/styles.ts

```diff
@@ -18,8 +28,15 @@ export const StyledUserContainer = styled('div', {
 
 export const StyledProjectContainer = styled('div', {
   width: '100%',
-  '@desktop': {
-    padding: '40px 100px',
+  '& > h4': {
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginTop: '5px',
+    justifyContent: 'center',
+    display: 'flex',
+    alignSelf: 'flex-start',
+    textAlign: 'center',
```

Comment:
I think you only need `textAlign: center` to center this text. Please check if the button has some unnecessary properties as well.